### PR TITLE
Project tokens creation/deletion

### PR DESCRIPTION
**Pull request by jcontonio**
* Adds ability to create/delete tokens
* Fixes endpoint listing when no token exists
* Adjusts DATA docs for clearer steps when needing to re-run migrations

### CODE REVIEW

File: web/src/client/js/actions/index.js [hunk 1]
```diff
@@ -88,6 +93,10 @@ export const ProjectAssignees = {
 export const ProjectTokens = {
   request: makeActionCreator(ActionTypes.REQUEST_PROJECT_TOKENS, 'projectId'),
   receive: makeActionCreator(ActionTypes.RECEIVE_PROJECT_TOKENS, 'projectId', 'data'),
+  create: makeActionCreator(ActionTypes.CREATE_PROJECT_TOKEN, 'projectId'),
+  receiveOneCreated: makeActionCreator(ActionTypes.RECEIVE_CREATED_PROJECT_TOKEN, 'projectId', 'data'),
```
Comment (jjidt) [inline web/src/client/js/actions/index.js]: the projectId is in the returned data. Do we need to pass the projectId in the action as well, or can reducers pull that out of the data object?

**Review by jjidt — CHANGES_REQUESTED** [review]
Looks/works great! just the one nitpick

**Comment by jcontonio** [comment]
All set, thanks @jjidt

**Review by jjidt — APPROVED** [review]
👍

**Pull request merged**